slackbot/Makefile

```diff
@@ -0,0 +1,68 @@
+.PHONY: help build push dev test lint type-check clean install
+
+# Default target
+help:
+	@echo "Kagent Slackbot - Available targets:"
+	@echo "  make install     - Install Python dependencies"
+	@echo "  make dev         - Run bot locally"
+	@echo "  make test        - Run tests (if available)"
+	@echo "  make lint        - Run linting with ruff"
+	@echo "  make type-check  - Run type checking with mypy"
+	@echo "  make build       - Build Docker image"
```

Comment:
`make build` doesn't work for me: 

<img width="1206" height="351" alt="Screenshot 2025-10-20 at 1 57 36 PM" src="https://github.com/user-attachments/assets/91465b65-5889-4198-8588-087c5bf9a7d9" />


Should there be a default config folder with some default permissions?